tests/tests/control

```diff
@@ -1,23 +1,21 @@
 ###############################################################################
 #
-#  FV3 CCPP Gaussian grid netcdf output test
+#  FV3 CCPP control test
```

Comment:
Do we still eed the `CCPP` in the comment and in the `TEST_DESCR`? This file and every other test in the directory.

Reply:
The test name is changed to:  global control test: GFSv16 atmosphere only at C96L127